### PR TITLE
Introduce ovirt-engine-repositories role

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
@mwperina  @machacekondra Just moved all files from other repo, and added the rpm related scripts (just copied the from vm-infra making changes where necessary)/

**Comment by KKoukiou** [comment]
Also can somebody with appropriate permissions enable the Travis CI for this repository?

**Comment by machacekondra** [comment]
ci build please

**Comment by machacekondra** [comment]
I've setup everything. Also in engine-setup repo. CI will start to work, once this patch is merged:

 https://gerrit.ovirt.org/#/c/85203/

**Comment by KKoukiou** [comment]
@machacekondra Super, thanks!

**Review by mwperina — APPROVED** [review]

File: build.sh
Comment (machacekondra) [inline build.sh]: VERSION="1.1.0"

File: build.sh
Comment (machacekondra) [inline build.sh]: MILESTONE=master

File: build.sh
Comment (machacekondra) [inline build.sh]: RPM_RELEASE="0.1.$MILESTONE.$(date -u +%Y%m%d%H%M%S)"

File: build.sh
Comment (machacekondra) [inline build.sh]: git ls-files | tar --files-from /proc/self/fd/0 -czf "$TARBALL" ovirt-ansible-repositories.spec

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: Summary: Ansible role to setup oVirt/RHV repositories

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: s/set/set up/

seems better for me, but I am not sure.

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: You can remove this `%pretrans`, it was used only for roles which existed in 4.1

**Comment by machacekondra** [comment]
Thanks for contributing automation files 👍

File: build.sh
Comment (machacekondra) [inline build.sh]: you can remove this line

**Comment by KKoukiou** [comment]
@mwperina @machacekondra Anything else I could change here? If not let's proceed with merging this one because tests for https://github.com/oVirt/ovirt-ansible-engine-setup/pull/1 are depending on this role.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for an this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organisation's projects.

Note: You might see multiple messages like this showing up on a single PR, this is due to a known bug in the GitHub trigger plugin on Jenkins, the infra team is working on fixing this via new testing flow in the near future, so we'll be able to post a single message per PR.

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by machacekondra** [comment]
ci add to whitelist

**Comment by machacekondra** [comment]
ci build please

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by ovirt-infra** [comment]
Some tests failed

**Comment by machacekondra** [comment]
ci add to whitelist

**Comment by machacekondra** [comment]
ci build please

**Review by mwperina — APPROVED** [review]

**Comment by machacekondra** [comment]
@mwperina Do we want to create legacy symlink for newly created roles? Or are we ok with name just `oVirt.repositories`?

**Comment by mwperina** [comment]
No, new roles (unreleased in oVirt 4.1.z) should have only the new name, ie `oVirt.repositories`

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: Please remove lines  22-28

File: build.sh
Comment (machacekondra) [inline build.sh]: Please remove this line.

File: build.sh
Comment (machacekondra) [inline build.sh]: Please remove line 43 and 44.

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: Please remove this line.

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: Please remove this line.

File: ovirt-ansible-repositories.spec.in
Comment (machacekondra) [inline ovirt-ansible-repositories.spec.in]: Please remove this line.

**Comment by ifireball** [comment]
ci test please

**Review by machacekondra — APPROVED** [review]

**Comment by machacekondra** [comment]
ci buid please

**Comment by KKoukiou** [comment]
Thanks for quick reviews!

**Pull request merged**
